src/QAT/qat/purr/backends/live.py

```diff
@@ -218,20 +222,9 @@ def _execute_on_hardware(self, sweep_iterator: SweepIterator, package: QatFile):
                         aq.mode == AcquireMode.INTEGRATOR
                     )
 
-            playback_results = self.model.control_hardware.start_playback(
-                repetitions=repetitions, repetition_time=repetition_time
-            )
-
-            for channel, aqs in aq_map.items():
-                if len(aqs) > 1:
-                    raise ValueError(
```

Comment:
why remove this check? This is currently a correct validation

Reply:
it's still there, just from another loop ^^